katalog/thanos/thanos-components/thanos-rule/thanos-rule.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: thanos-rule
+  name: thanos-rule
+spec:
+  ports:
+    - port: 10902
+      protocol: TCP
+      targetPort: http
+      name: http-rule
+    - port: 10901
+      protocol: TCP
+      targetPort: grpc
+      name: grpc-rule
+  selector:
+    app: thanos-rule
+  sessionAffinity: None
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: thanos-rule
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: thanos-rule
+  labels:
+    app: thanos-rule
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: thanos-rule
+  template:
+    metadata:
+      labels:
+        app: thanos-rule
+    spec:
+      serviceAccountName: thanos-rule
+      containers:
+        - name: thanos-rule
+          image: thanos
+          args:
+            - rule
+            - --log.level=info
+            - --data-dir=/var/thanos/data/
+            - --rule-file=/var/thanos/rules/*.yaml
+            - --alertmanagers.url=$(ALERTMANAGER_URL)
+            - --eval-interval=30s
+            - --alert.query-url=$(THANOS_QUERY_URL)
+            - --query.sd-files=/etc/thanos/query-sd.yaml
+          env:
+            - name: ALERTMANAGER_URL
+              value: "dns+http://alertmanager.base-namespace"
```

Comment:
Our default alertmanager setup is deployed into the monitoring namespace. Is this value right configured?

Reply:
right @angelbarrera92 , we are not using thanos-rule, we have added it because is one of optional components that Thanos provides (as you can see is not added in the kustomization.yaml) . But, if someone need to use it, for sure need to patch this values accordingly with service names.